### ISSUE TITLE
Fitting my continuous data with hmmlearn GMMHMM library in python

### ISSUE THREAD

**Issue by eponraj27392** [hmmlearn/hmmlearn#343]
Hello all,
I am trying to implement HMM Learning/training with the continuous data using hmmlearn library in python.

**Overview about my Data set**

For E.g, A list contains normalized training data.

Total Elements inside the list = 75

Each element inside the list is an np array.

The data inside the list is Vehicle driving data which is continuous.

```
'X_train = [[100*4], [100*4], [100*4], [100*4].........................[100*4]]' 
len(X_train) = 75
X_Train[0] = [100*4]
X_Train.columns=['Speed','Angle', 'Acceleration1', 'Acceleration2']
```

**Note**: Every `[100*4]` data received from vehicle at specific Time intervals. Lets say `X_train[0]` is 15 to 30 seconds driving study data. `X_train[1.]` may be 15 to 30 seconds driving study data and so on .....

**hmmlearn problem facing**

In hmmlearn library i would like to clarify some queries. My objective is to get the parameters of the trained model for my dataset like intial prob, state transition prob, Gaussian weights, mean & covariance, so,.

**Model created with 2 Hidden states with 2 Gaussian**
model = hmm.GMMHMM(n_components=2, n_mix=2, covariance_type="diag", n_iter=100)

Question :1 whether i should concatenate my overall data set into a single list as stated in the document [Working with multiple seq from hmmlearn document](https://hmmlearn.readthedocs.io/en/latest/tutorial.html#multiple-sequences) ?

If it is, then my t`raining data set (X)` will look like with size of `7500 *4`, also, the lengths parameter will have a list like `lengths = [100, 100, 100, ......100]`

**Is this right ? Is this the way fit should be implemented ? Kindly correct me, If i am wrong**

**Question : 2** How can i check my `EM Algo converges` ? I have seen the document for monitor_ function. But I cannot able to understand. Kindly explain how to apply it.

**Question : 3** After training, i would like to check the probability `(using Forward Algorithm) `of non-trained sequence of `size 100*4`. So what is the required function to apply forward algorithm using trained model parameters ?

Kindly support. I have posted many question related to HMM training, But I have no satisfied answers based on my requirement. I hope hmmlearn library helps

I have posted this question in stack too..
https://stackoverflow.com/questions/56500854/fitting-my-continuous-data-with-hmmlearn-gmmhmm-library-in-python

**Comment by eponraj27392** [comment]
@superbobry  I hope, you can answer my question

**Comment by eponraj27392** [comment]
Whether the question is not relevant to this page, since I am not getting any answers or comments ?
Any one here ?

**Comment by anntzer** [comment]
@fcostin has kindly provided a fix to fit multiple sequences using GMMHMM, which was previously not supported (https://github.com/hmmlearn/hmmlearn/issues/346#issuecomment-875901741).  This is now released as hmmlearn 0.2.6.  Please consider upgrading to it, and reopen the issue if your problem is still present.
